web/themes/new_weather_theme/templates/partials/radar.html.twig

```diff
@@ -21,10 +21,22 @@
         });
       </script>
     </div>
-    <div class="grid-row wx-radar-container">
-       <wx-radar class="grid-col-12" lat="{{ point.lat }}" lon="{{ point.lon }}">
-         <div id="wx_radar_container" style="height: 300px;"></div>
+    <div class="grid-row wx-radar-container position-relative">
+      <wx-radar class="grid-col-12" lat="{{ point.lat }}" lon="{{ point.lon }}">
+        <div id="wx_radar_container" style="height: 300px;"></div>
       </wx-radar>
+
+      <div class="position-absolute top-0 right-0 height-full">
+        <div class="position-sticky top-0" style="top: 55px;">
```

Comment:
Just playing around with this, I'm wondering if there needs to be the outer `position-absolute` div at all. I'm thinking you could just have the inner one, have _it_ be positioned absolutely, and give it top and right values of like 5px or something. That might accomplish the same thing without all the worry of `sticky` and you won't need the inline style attribute at all -- you could get rid of that code in the js file
```suggestion
        <div class="position-absolute top-2px right-2px">
```

Reply:
(`top-0` and `right-0` will also work, due to the positioning of the SVG within the containing div there is already some margin)